contrib/dependencies/yum-packages.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+yum install -y --allowerasing \
```

Comment:
Switching to dnf would make sense. Then also the file should be renamed. So maybe something for another PR.